test/integration/targets/aws_lambda/tasks/main.yml

```diff
@@ -0,0 +1,381 @@
+---
```

Comment:
I am putting an alias in for this as "lambda" which I think will have the same effect.  These tests cover both lambda and execute_lambda.  Maybe we should instead speed up the rename.  I don't think lambda is that heavilly used now and we may be hoping use will increase soon.